### PR TITLE
SMA Weapons compatibility

### PR DESCRIPTION
**Pull request by Kwinno**
**When merged this pull request will:**
- Adds compatibility pbo for SMA Weapons by Blazenchamber (ACE Ammo and HuntIR)

### CODE REVIEW

File: optionals/compat_sma_weapons/script_component.hpp [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#define COMPONENT sma_weapons
+#define COMPONENT_BEAUTIFIED Blazenchamber SMA Weapons Mod Compatibility
```
Comment (jonpas) [inline optionals/compat_sma_weapons/script_component.hpp]: Just `SMA Weapons Compatibility` should be fine.

**Comment by LecksC** [comment]
This likely needs to be updated for SMA 2.7. Class names can be found here http://smaproject.com/support/

**Comment by Kwinno** [comment]
I'm aware @LecksC trying to find the time to go over everything. since it was pushed to 3.9 i didn't rush it

**Comment by bux578** [comment]
^ That's another superb reason to have the compat pbos as a separate repo.

**Comment by Kwinno** [comment]
@bux578 you mean so releases can go with mod releases and not ace releases?

**Comment by bux578** [comment]
@Kwinno yes.

**Comment by WastedMike** [comment]
Is this up to date?

**Pull request closed without merging**